dts/arm/st/g0/stm32g0.dtsi

```diff
@@ -198,6 +198,28 @@
 			status = "disabled";
 			label = "I2C_2";
 		};
+
+		spi1: spi@40013000 {
+			compatible = "st,stm32-spi-fifo", "st,stm32-spi";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x40013000 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_APB2 0x00001000>;
+			interrupts = <25 0>;
+			status = "disabled";
+			label = "SPI_1";
+		};
+
+		spi2: spi@40003800 {
+			compatible = "st,stm32-spi";
```

Comment:
```suggestion
			compatible = "st,stm32-spi-fifo", "st,stm32-spi";
```

Reply:
I have added this with the latest commits.